### DISCUSSION TITLE
Error while trying Fun Features

### DISCUSSION THREAD

**Discussion by LintStalker** [YimMenu/YimMenu#1735] [General]
While in the Fun Features section of the menu, the following error started coming up in the Yimmenu logging screen:

 WARN/Inovker.cpp:56 Exception caught while trying to call a long hexadecimal number

Once I left that screen, it stopped.  

I was using last night's build.

**Comment by LintStalker** [comment]
Ok, I must have downloaded the file just before it was updated

**Reply by holysnipz** [reply]
to be fair, nightly release build has not yet updated to recent commit that have that issue fixed

either grab artifact from [CI](https://github.com/YimMenu/YimMenu/actions/workflows/ci.yml), or build it yourself

**Comment by LintStalker** [comment]
Ok, so that explains why still saw the error.  No big deal, I just wanted
to let folks know about it
